lucene/analysis/common/src/java/org/apache/lucene/analysis/hunspell/Stemmer.java

```diff
@@ -259,12 +258,8 @@ boolean doStem(
         }
       }
     }
-    try {
-      return stem(
-          word, offset, length, context, -1, Dictionary.FLAG_UNSET, -1, 0, true, false, processor);
-    } catch (IOException bogus) {
-      throw new RuntimeException(bogus);
-    }
+    return stem(
```

Comment:
catching `IOException` in `nextArc` allows us to get rid of declaring it everywhere